client/src/UpdateForm.js

```diff
@@ -1,97 +0,0 @@
-import React, { Component } from "react";
-import "./App.css";
-import axios from "axios";
-
-const BASE_URL = "http://localhost:3005";
-
-class UpdateForm extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      name: "",
-      image_url: "",
-      scary: {
-        level: 1
-      }
-    };
-  }
-  handleChange = (event, key) => {
-    this.setState({
-      [key]: event.target.value
-    });
-  };
-
-  handleScaryLevel = event => {
-    const { scary } = this.state.scary;
-    const newScary = Object.assign({}, scary);
-    newScary.level = event.target.value;
-    this.setState({ scary: newScary });
-  };
```

Comment:
good job with this function. here's another way you could write this with a little less code: 
```
handleScaryLevel = event => {
  this.setState({ scary: { level: event.target.value }})
}
```
However, if the `scary` object had other properties on it, then something like your code would be the way to go.